pyrq/pools.py

```diff
@@ -0,0 +1,226 @@
+import time
+import socket
+import os
+
+from pyrq import helpers
+
+DEFAULT_CHUNK_SIZE = 10
```

Comment:
navíc v php implementaci je možnost nastavit to separátně pro jednotlivé operace